.changeset/rich-dolls-compete.md

```diff
@@ -0,0 +1,34 @@
+---
+"astro": minor
+---
+
+Improves DX around `500.astro`
+
+The special `src/pages/500.astro` page now accepts an error as a prop. It can be anything so make sure to handle it properly:
```

Comment:
```suggestion
Improves the developer experience of the `500.astro` file.

The special `src/pages/500.astro` page now accepts an error as a prop. It can be anything, so make sure to handle it properly:
```